include/SDL3/SDL_platform.h

```diff
@@ -22,191 +22,12 @@
 /**
  *  \file SDL_platform.h
  *
- *  Try to get a standard set of platform defines.
+ *  Header file for platform functions.
  */
 
 #ifndef SDL_platform_h_
 #define SDL_platform_h_
 
-#if defined(_AIX)
```

Comment:
We should replace this block with:
`#include <SDL3/SDL_platform_defines.h>`

Reply:
Ah yes, then for users of SDL there is no more visible difference.